### PR TITLE
Fix the warning log when edgemesh is disabled

### LINKED ISSUE #2597: A large number of warning logs when edgemesh is disabled
**What happened**:
A large number of warning logs when edgemesh is disabled

> W0303 14:00:55.553796 3106 context_channel.go:295] Failed to get channel, type:edgemesh
W0303 14:00:55.553837 3106 context_channel.go:72] Get bad module name :edgemesh when send message, do nothing
W0303 14:00:55.595438 3106 context_channel.go:295] Failed to get channel, type:edgemesh
W0303 14:00:55.595475 3106 context_channel.go:72] Get bad module name :edgemesh when send message, do nothing
W0303 14:00:57.572297 3106 context_channel.go:295] Failed to get channel, type:edgemesh
W0303 14:00:57.572341 3106 context_channel.go:72] Get bad module name :edgemesh when send message, do nothing
W0303 14:00:57.615176 3106 context_channel.go:295] Failed to get channel, type:edgemesh
W0303 14:00:57.615211 3106 context_channel.go:72] Get bad module name :edgemesh when send message, do nothing

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- KubeEdge version(e.g. `cloudcore --version` and `edgecore --version`):
- <details><summary>Cloud nodes Environment:</summary>

  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>
- <details><summary>Edge nodes Environment:</summary>

  - edgecore version (e.g. `edgecore --version`):
  - Hardware configuration (e.g. `lscpu`):
  - OS (e.g. `cat /etc/os-release`):
  - Kernel (e.g. `uname -a`):
  - Go version (e.g. `go version`):
  - Others:

  </details>

### PR DESCRIPTION
**Pull request by hackers365**
Signed-off-by: hackers365 <hackers365@gmail.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
check whether the edgeMesh module is enabled in the sendToEdgeMesh function

**Which issue(s) this PR fixes**:

Fixes #2597 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/metamanager/process.go [hunk 1]
```diff
@@ -60,6 +61,10 @@ func sendToEdged(message *model.Message, sync bool) {
 }
 
 func sendToEdgeMesh(message *model.Message, sync bool) {
+	//if edgeMesh module disabled, then return
```
Comment (fisherxu) [inline edge/pkg/metamanager/process.go:64]:
```suggestion
	// if edgeMesh module disabled, then return
```

**Review by fisherxu — COMMENTED** [review]
lgtm

**Comment by fisherxu** [comment]
@hackers365  Please rebase the master branch to fix ci issue :)

**Comment by fisherxu** [comment]
And you'd better to squash the commits :)

**Comment by fisherxu** [comment]
ping @hackers365  :)

**Comment by kevin-wangzefeng** [comment]
Seems the travis checking is failing due to time out.
Close reopening

**Comment by fisherxu** [comment]
@hackers365 Could you please cherry pick this PT to release-1.6 branch using`hack/cherry_pick_pull.sh`?

**Pull request merged**